### ISSUE TITLE
[BUG] System.IO.IOException: Access to the path '\\?\C:\Users\User\AppData\Local\Bloxstrap\Roblox\Player\content\textures\Cursors\KeyboardMouse' is denied.

### ISSUE THREAD

**Issue by DesignerTea** [bloxstraplabs/bloxstrap#3483]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.0

### What problem did you encounter?

[BUG] System.IO.IOException: Access to the path '\\?\C:\Users\User\AppData\Local\Bloxstrap\Roblox\Player\content\textures\Cursors\KeyboardMouse' is denied.
![image](https://github.com/user-attachments/assets/f6eade35-bbf0-4f93-a9d4-8f5a8472a7e7)
![image_2024-10-26_130028490](https://github.com/user-attachments/assets/be857059-5b6c-46d8-961d-6809686eafd2)
how can i fix it?

### Bloxstrap Log

```text
2024-10-26T09:53:25Z [App::OnStartup] Starting Bloxstrap v2.8.0
2024-10-26T09:53:25Z [App::OnStartup] Compiled Wednesday, 16 October 2024 at 12:44:37 PM from commit 42ffec26cf2e080fe22210305a08fcb286aeb577 (tag/v2.8.0)
2024-10-26T09:53:25Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-10-26T09:53:25Z [App::OnStartup] Temp path is %UserProfile%\AppData\Local\Temp\Bloxstrap
2024-10-26T09:53:25Z [App::OnStartup] WindowsStartMenu path is %UserProfile%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
2024-10-26T09:53:25Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20241026T095325Z.log
2024-10-26T09:53:25Z [Logger::Initialize] Finished initializing!
2024-10-26T09:53:25Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-10-26T09:53:25Z [JsonManager<Settings>::Load] Loaded successfully!
2024-10-26T09:53:25Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-10-26T09:53:25Z [JsonManager<State>::Load] Loaded successfully!
2024-10-26T09:53:25Z [FastFlagManager::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-10-26T09:53:25Z [FastFlagManager::Load] Loaded successfully!
2024-10-26T09:53:26Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper
2024-10-26T09:53:26Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper dialog
2024-10-26T09:53:26Z [Bootstrapper::Run] Running bootstrapper
2024-10-26T09:53:26Z [Bootstrapper::SetStatus] Connecting to Roblox...
2024-10-26T09:53:26Z [Deployment::TestConnection<https://setup.rbxcdn.com>] Connecting...
2024-10-26T09:53:26Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/versionStudio
2024-10-26T09:53:26Z [Deployment::InitializeConnectivity] Testing connectivity...
2024-10-26T09:53:27Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/versionStudio
2024-10-26T09:53:27Z [Deployment::InitializeConnectivity] Got https://setup.rbxcdn.com as the optimal base URL
2024-10-26T09:53:27Z [Bootstrapper::Run] Connectivity check finished
2024-10-26T09:53:27Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-10-26T09:53:27Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-26T09:53:27Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-26T09:53:27Z [Bootstrapper::CheckForUpdates] No updates found
2024-10-26T09:53:27Z [Bootstrapper::GetLatestVersionInfo] Got channel as production
2024-10-26T09:53:27Z [Deployment::GetInfo] Getting deploy info for channel production
2024-10-26T09:53:27Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-26T09:53:28Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-26T09:53:28Z [RegistryKeyEx::SetValueSafe] Writing '' to HKEY_CURRENT_USER\SOFTWARE\ROBLOX Corporation\Environments\RobloxPlayer\Channel\www.roblox.com
2024-10-26T09:53:28Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/version-4ffdeb3e393e469e-rbxPkgManifest.txt
2024-10-26T09:53:28Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/version-4ffdeb3e393e469e-rbxPkgManifest.txt
2024-10-26T09:53:28Z [Bootstrapper::SetStatus] Installing {product}...
2024-10-26T09:53:28Z [LaunchHandler::LaunchRoblox] Bootstrapper task has finished
2024-10-26T09:53:28Z [LaunchHandler::LaunchRoblox] An exception occurred when running the bootstrapper
2024-10-26T09:53:28Z [App::FinalizeExceptionHandling] System.IO.IOException: Access to the path '\\?\%UserProfile%\AppData\Local\Bloxstrap\Roblox\Player\content\textures\Cursors\KeyboardMouse' is denied.
   at System.IO.FileSystem.RemoveDirectoryRecursive(String fullPath, WIN32_FIND_DATA& findData, Boolean topLevel)
   at System.IO.FileSystem.RemoveDirectory(String fullPath, Boolean recursive)
   at Bloxstrap.Bootstrapper.UpgradeRoblox()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.Bootstrapper.Run()
```

**Comment by pizzaboxer** [comment]
Fixed in 2.8.1 (assuming so, reopen if not)

**Comment by DesignerTea** [comment]
> Fixed in 2.8.1 (assuming so, reopen if not)
**tried to reinstall on the discord and its still havent work for me**
